### ISSUE TITLE
[Bug] Hash Facade is not imported

### ISSUE THREAD

**Issue by Naoray** [laravel-shift/factory-generator#3]
When generating a factory for a Model with a `password` column it will generate the following column in the factory file:

```php
'password' => Hash::make('password'),
```

We need to add the import statement at the top or let the developer know that he hast to add it.

```php
use Illuminate\Support\Facades\Hash;
```

**Comment by jasonmccreary** [comment]
Agreed. I noticed this myself the other day. We'll need to loop through the definition and check for common things to import, i.e. `Str` and `Hash`.
